src/site/content/en/blog/preconnect-and-dns-prefetch/index.md

```diff
@@ -0,0 +1,141 @@
+---
+layout: post
```

Comment:
where does `layout: post` come from? I can't find this yaml property in the template or docs

Reply:
I copied it from one of my earlier posts, it used to be in the example draft I think. Removed it now, it looks like it doesn't make a difference.

Reply:
My bad it's totally in the handbook https://web.dev/handbook/markup-post-codelab/ sorry